types/jsonld/index.d.ts

```diff
@@ -5,11 +5,15 @@
  */
 
 declare module 'jsonld' {
-  type CallbackFn = (err: null|Error, result?: any) => void
+  interface RemoteDocument {
+    contextUrl?: string;
+    documentUrl?: string;
+    document: unknown;
+  }
 
   interface JsonldOptions {
-    documentLoader: (url: string, callback: CallbackFn) => void
+    documentLoader: (url: string) => Promise<RemoteDocument>;
   }
 
-  export function expand(object: any, options: JsonldOptions): Promise<any>;
+  export function expand(object: unknown, options: JsonldOptions): Promise<any>;
```

Comment:
there are definitelytyped types for `jsonld` now, but `expand()` still has the old callback-based `documentLoader`, so easier to still just maintain types for our own tiny corner of the library. It would be nice to eventually switch and not rely on `LH.StructuredData.ExpandedSchemaRepresentation` being the source of truth, though.